tools/measure_coverage.sh

```diff
@@ -145,8 +130,7 @@ mv src/cover_js src/js
 tools/build.py --jerry-cmake-param="-DFEATURE_SYSTEM_ALLOCATOR=ON" \
     --target-arch=x86 --compile-flag="-coverage" --no-snapshot --no-check-test
 
-# Run tests
-build/i686-linux/debug/bin/iotjs tools/check_test.js -- output-coverage=yes
+python tools/testrunner.py ${PWD}/build/i686-linux/debug/bin/iotjs --quiet
```

Comment:
In order to get coverage information for peripheral I/O module such as gpio, we need to modify the above two lines manually. Do you have an idea on how to handle such a bit annoying stuff more easily?

Reply:
What if we add a parameter to `measure_coverage.sh`?

e.g.
```bash
testrunner=$1

if [ $testrunner = 'jsdriver' ];
then
  build/i686-linux/debug/bin/iotjs tools/check_test.js -- output-coverage=yes
elif [ $testrunner = 'pydriver' ];
then
  python tools/testrunner.py ${PWD}/build/i686-linux/debug/bin/iotjs --quiet
else
  echo "wrong testrunner argument"
  exit 1
fi
```